Helper.py

```diff
@@ -1,16 +1,16 @@
 import ROOT
 
 PI=3.14159

```

Comment:
In this function you should also pass the year as the list of triggers. As the list of triggers depends on year.

also instead of many if else add all triggers in the array and then use it.